assets/training/distillation/components/data_generation/src/common/utils.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Data generator utils."""
+
+import os
+import time
+
+from azure.ai.ml import MLClient
+from azure.ai.ml.identity import AzureMLOnBehalfOfCredential
+from azure.identity import AzureCliCredential, ManagedIdentityCredential
+from azureml.acft.common_components import get_logger_app
+from azureml.core import Run, Workspace
+from azureml.core.run import _OfflineRun
+
+from typing import Union
+
+
+logger = get_logger_app("azureml.acft.contrib.hf.nlp.entry_point.data_import.data_import")
```

Comment:
We might want to change this as well? 

Also, I am seeing logs being scrubbed. How do we make sure they are pushed out?